### PR TITLE
Blogpost: 4 Tools to Deliver High-Quality Lambda Functions

### PR DESCRIPTION
**Pull request by taavirehemagi**
Hey guys!

I thought I would share my experience about what tooling comes the most handy and works the best alongside serverless to develop lambda based systems. 

Let me know what you think and if you need me to change anything.

### CODE REVIEW

**Comment by worldsoup** [comment]
this is a great post Taavi, I think our audience will find it really interesting, thanks for sharing! I made a minor change to the title and am going to schedule it to go live along with tweets and mentions in our newsletter on 6/12 (Monday). Let me know if you have any questions! nick@serverless.com

**Comment by taavirehemagi** [comment]
That's awesome Nick! Thank you

File: posts/2017-05-30-4-tools-to-deliver-high-quality-lambda-functions.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-title: 4 Tools to Deliver High-Quality Lambda Functions
+title: 4 Tools For Building Production Serverless Arcitectures with Lambda
```
Comment (taavirehemagi) [inline posts/2017-05-30-4-tools-to-deliver-high-quality-lambda-functions.md]: I think you have a typo in `Architectures` :)

Comment (worldsoup) [inline posts/2017-05-30-4-tools-to-deliver-high-quality-lambda-functions.md]: good catch :)

**Comment by worldsoup** [comment]
schedule(06/12/2017 6:00 am)

**Comment by DavidWells** [comment]
✅ Post scheduled for **06/12/2017 at 6:00am (America/Los_Angeles)**

**To unscheduled the post:**

1. **[Delete the previous comment ⬆](https://github.com/serverless/blog/pull/187#issuecomment-306888789)** with schedule(time) information by clicking the ❌
  ```
  ┌──────────────────────────────────────────────┐
  │                                           X  │
  ├───────────────────────────────────────────▲──┤
  │                                           │  │
  │  schedule(mm/dd/yyyy h:mm A)   ┌────────┐ │  │
  │                                │ Delete ├─┘  │
  │                                └────────┘    │
  └──────────────────────────────────────────────┘
  ```
2. (Optionally) Delete this comment to keep things tidy

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Pull request merged**